jacobian_follower/src/jacobian_follower/bindings.cpp

```diff
@@ -42,5 +42,12 @@ PYBIND11_MODULE(pyjacobian_follower, m)
            py::arg("grippers"),
            py::arg("max_velocity_scaling_factor"),
            py::arg("max_acceleration_scaling_factor")
-      );
+      )
+      .def("compute_IK_solutions", &JacobianFollower::compute_IK_solutions,
+           py::arg("pose"),
+           py::arg("joint_group_name"))
+      .def("fk", &JacobianFollower::computeFK,
+       py::arg("joint_angles"),
+       py::arg("joint_group_name"))
+           ;
 }
```

Comment:
newline error here?

Reply:
What newline error?

Reply:
huh idk, there was a red :no_entry:  symbol here before

Reply:
wait no it's still there. I think it means there is no newline at the end of the file?

Reply:
Should be fixed in the updated PR